frontend/src/LandingPage.jsx

```diff
@@ -1,12 +1,13 @@
-import React, { useEffect } from 'react'; 
+import React, { useEffect } from 'react';
 import { useState } from 'react';
 import { Link } from 'react-router-dom';
 import { makeStyles } from '@mui/styles';
 import Box from '@mui/material/Box';
 import Button from '@mui/material/Button';
 import Modal from '@mui/material/Modal';
-import Typography from '@mui/material/Typography'; 
-import ReportDashboard from './components/ReportDashboard'; 
+import Typography from '@mui/material/Typography';
+import ReportDashboard from './components/ReportDashboard';
```

Comment:
What changed in the above 2 lines of code? I don't see it

Reply:
Sorry for the late reply, there are no changes in this code here. I am unsure why it shows there are some.